### ISSUE TITLE
Can we run SoloX in docker?

### ISSUE THREAD

**Issue by Sivasankaramalan** [smart-test-ti/SoloX#163]

**Comment by rafa0128** [comment]
Please check if your mobile device has turned on the debug switch?

**Comment by Sivasankaramalan** [comment]
Yes, When i'm running soloX in local it was working perfectly fine, but while running the service in docker, it is not able to get the device details, 

OS : MAC
